### PR TITLE
Simple shape renderer

### PR DESCRIPTION
**Pull request by gmaker**

### CODE REVIEW

File: core/src/com/ooutofmind/gfx/Art.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    private Art() {
+    }
+
+    public static void circle(int x, int y, int r, Color color, ShapeRenderer shapeRenderer) {
+        shapeRenderer.setColor(color);
+        shapeRenderer.circle(x, y, r);
+    }
+
+    public static void quad(int x, int y, int w, int h, int border, Color color, ShapeRenderer shapeRenderer) {
```
Comment (ooutofmind) [inline core/src/com/ooutofmind/gfx/Art.java:18]: did you mean square or rectangle?

File: core/src/com/ooutofmind/Reg.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import com.badlogic.gdx.utils.Disposable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Reg implements Disposable {
+    private static final List<Disposable> toBeRemoved = new ArrayList<>();
+
+    public static final Reg i = new Reg();
```
Comment (ooutofmind) [inline core/src/com/ooutofmind/Reg.java:11]: instance?

**Review by ooutofmind — APPROVED** [review]

File: core/src/com/ooutofmind/gfx/Art.java (hunk 1 above)
Comment (gmaker) [reply core/src/com/ooutofmind/gfx/Art.java:18]: rect or quad

File: core/src/com/ooutofmind/Reg.java (hunk 2 above)
Comment (gmaker) [reply core/src/com/ooutofmind/Reg.java:11]: nope ;) welcome to fucking gamedev prototyping

**Pull request merged**